kubeadm/scripts/Install-Containerd.ps1

```diff
@@ -1,90 +0,0 @@
-<#
-.SYNOPSIS
-Installs ContainerD on a Windows machines in preperation for joining the node to a Kubernetes cluster.
```

Comment:
Should we not move this script as well? I believe this is still needed to install ContainerD on Windows

Reply:
yes, good catch!

Reply:
fixed